### PR TITLE
Results Buttons

### LINKED ISSUE #342: Strength and deformation result view buttons
Create the buttons to activate the deformation and strength views per the mock-ups.

### PR DESCRIPTION
**Pull request by jake-kirkland-teton**
Added new buttons for prepping new results system and additional messages.

Let me know of the opacity needs changed.

Closes #342

### CODE REVIEW

**Review by dougkenik — CHANGES_REQUESTED** [review]
Functionality looks good - just the comments

File: SmartSlicePlugin/stage/ui/SmartSliceResultsButtons.qml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        anchors.fill: parent
+        height: childrenRect.height
+        width: childrenRect.width
+
+        Row {
+            spacing: UM.Theme.getSize("default_margin").width
+
+            anchors.horizontalCenter: parent.horizontalCenter
+
+            UM.SimpleButton {
```
[the remark this answers is not in the document]
Comment (jake-kirkland-teton) [reply SmartSlicePlugin/stage/ui/SmartSliceResultsButtons.qml:51]: Ah yeah I had meant to put a hover color on these and spaced it. I will do that. Will less opaque work for that?

And this basically is a recolor button. This uses a RecolorImage as the icon and adds button functionality.

File: SmartSlicePlugin/stage/ui/SmartSliceResultsButtons.qml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    height: UM.Theme.getSize("action_button").height + (UM.Theme.getSize("thick_margin").height * 2)
+
+    anchors {
+        rightMargin: UM.Theme.getSize("thick_margin").width
+        bottomMargin: UM.Theme.getSize("thick_margin").height
+        bottom: parent.bottom
+    }
+
+    color: UM.Theme.getColor("main_background")
+    border.width: UM.Theme.getSize("default_lining").width
```
Comment (dougkenik) [inline SmartSlicePlugin/stage/ui/SmartSliceResultsButtons.qml:22]: Let's also use the {} with border

**Review by dougkenik — APPROVED** [review]
You did a very nice job with this. Looks great

File: SmartSlicePlugin/stage/ui/SmartSliceResultsButtons.qml (hunk 2 above)
Comment (jake-kirkland-teton) [reply SmartSlicePlugin/stage/ui/SmartSliceResultsButtons.qml]: Ah good call. And thanks!

**Review by bradymadams — APPROVED** [review]
Looks good :+1:

**Pull request merged**